pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/kube_ingress_reconciler.go

```diff
@@ -220,11 +220,7 @@ func createRawIngress(scheme *runtime.Scheme, isvc *v1beta1.InferenceService,
 			return nil, fmt.Errorf("failed creating explainer ingress host: %v", err)
 		}
 		// :predict routes to the predictor when there is only predictor and explainer
-		if len(isvc.Spec.Predictor.Containers) != 0 && len(isvc.Spec.Predictor.Containers[0].Ports) != 0 {
-			rules = append(rules, generateRule(host, predictorName, "/", isvc.Spec.Predictor.Containers[0].Ports[0].ContainerPort))
-		} else {
-			rules = append(rules, generateRule(host, predictorName, "/", constants.CommonDefaultHttpPort))
-		}
+		rules = append(rules, generateRule(host, predictorName, "/", constants.CommonDefaultHttpPort))
```

Comment:
I see so `if len(isvc.Spec.Predictor.Containers) != 0 && len(isvc.Spec.Predictor.Containers[0].Ports) != 0` only works for custom predictor, for standard predictor you would need to check the ports on the `model` spec.

Reply:
indeed